extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -968,14 +967,14 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 					case NetworkContainerType.NETWORK_SHARE:
 						requestBody.properties.backupConfiguration = {
 							targetLocation: {
-								storageAccountResourceId: this._databaseBackup.networkShare.storageAccount.id,
-								accountKey: this._databaseBackup.networkShare.storageKey,
+								storageAccountResourceId: this._databaseBackup.networkShares[i].storageAccount.id,
+								accountKey: this._databaseBackup.networkShares[i].storageKey,
 							},
 							sourceLocation: {
 								fileShare: {
-									path: this._databaseBackup.networkShare.networkShareLocation,
-									username: this._databaseBackup.networkShare.windowsUser,
-									password: this._databaseBackup.networkShare.password,
+									path: this._databaseBackup.networkShares[i].networkShareLocation,
+									username: this._databaseBackup.networkShares[i].windowsUser,
+									password: this._databaseBackup.networkShares[i].password,
```

Comment:
can you make sure this password change doesn't get saved to the migration status in global storage and verify it doesn't show up in the 'copy migration details' content

Reply:
Password is an empty string in "Copy migration details ":
```
"sourceLocation": {
          "fileShare": {
            "path": "\\\\aalab03-2k8\\SharedBackup\\aasim-test",
            "username": "AALAB03-2K8\\hijavatestlocaluser",
            "password": ""
          }
        }
```